### PR TITLE
Renamed "calinski_harabaz_score" to "calinski_harabasz_score" (added 's' in harabasz)

### PR DESCRIPTION
**Pull request by thisHermit**
This PR fixes the DeprecationWarning on using 'calinski_harabaz_score' and replaces it with the correct spelling and attempts to increase the progress for #318 by using PEP8. This also fixes the test case.

`DeprecationWarning: Function calinski_harabaz_score is deprecated; Function 'calinski_harabaz_score' has been renamed to 'calinski_harabasz_score' and will be removed in version 0.23.`

I have changed the following:

1. Renamed 'test_calinski_harabaz_metric.png' to 'test_calinski_harabasz_metric.png'
2. Renamed test case to test_calinski_harabasz_metric in test_elbow.py
3. Renamed all instances of calinski_harabaz to calinski_harabasz

### TODOs and questions

Questions for the @DistrictDataLabs/@rebeccabilbro:

- [ ] Should I continue removing DeprecationWarnings?

### CODE REVIEW

**Comment by thisHermit** [comment]
Hi @Kautumn06. Sorry for forgetting the fix for the documentation. The only thing left to do is fixing the bit.ly link to the https://scikit-learn.org/stable/modules/generated/sklearn.metrics.calinski_harabasz_score.html in yellowbrick/cluster/elbow.py, line number 226.

**Comment by thisHermit** [comment]
Hi @Kautumn06. Sorry for forgetting the fix for the documentation. The only thing left to do is fixing the bit.ly link to the https://scikit-learn.org/stable/modules/generated/sklearn.metrics.calinski_harabasz_score.html in yellowbrick/cluster/elbow.py, line number 226.

**Comment by bbengfort** [comment]
Hi @thisHermit here is the bit.ly link for the correct metric:

http://bit.ly/2ItAgts

**Comment by thisHermit** [comment]
@bbengfort, @Kautumn06, I hope I have fixed all issues now.

**Comment by Kautumn06** [comment]
Thank you @thisHermit for also updating the method name in the docs! I'll look over the changes today and once the tests pass, merge this in. I really appreciate your work on this and thank you again for contributing to Yellowbrick!
